### PR TITLE
Add a compute concept modal.

### PR DESCRIPTION
**Pull request by nsthorat**
Towards https://github.com/lilacai/lilac/issues/643

Add a compute concept modal.

https://github.com/lilacai/lilac/assets/1100749/f64fb732-4819-4192-99e0-75f8aef7fcb3

### CODE REVIEW

File: web/blueprint/src/lib/components/commands/selectors/ConceptList.svelte [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  const concepts = queryConcepts();
+  const authInfo = queryAuthInfo();
+  $: userId = $authInfo.data?.user?.id;
+  $: username = $authInfo.data?.user?.given_name;
+
+  $: {
+    if ($concepts.data != null) {
+      selectedConceptInfo = $concepts.data[0];
+    }
+  }
```
Comment (dsmilkov) [inline web/blueprint/src/lib/components/commands/selectors/ConceptList.svelte:20]: tip: you could shorten this to and avoid line 9
```
$: selectedConceptInfo = $concepts.data?.[0];
```

File: web/blueprint/src/lib/components/commands/CommandConcepts.svelte
Comment (dsmilkov) [inline web/blueprint/src/lib/components/commands/CommandConcepts.svelte:1]: Replace the long random number user id shown in the video on the LHS of the modal with the name of the logged in person?

**Review by dsmilkov — APPROVED** [review]
Awesome

File: lilac/signal.py [hunk 2]
```diff
@@ -176,7 +177,34 @@ def fields(self) -> Field:
 
 class VectorSignal(Signal, abc.ABC):
   """An interface for signals that can compute items given vector inputs."""
-  embedding: str
+  embedding: str = PydanticField(description='The name of the pre-computed embedding.')
+
+  class Config:
+    underscore_attrs_are_private = True
```
[the remark this answers is not in the document]
Comment (nsthorat) [reply lilac/signal.py]: Done

File: web/lib/src/lilac.ts [hunk 3]
```diff
@@ -225,6 +225,13 @@ export function getSignalInfo(field: LilacField): Signal | null {
   return null;
 }
 
+/** Find the computed embeddings for a field. */
+export function getComputedEmbeddings(field: LilacField): LilacField[] {
```
[the remark this answers is not in the document]
Comment (nsthorat) [reply web/lib/src/lilac.ts]: Done

File: web/blueprint/src/lib/components/commands/selectors/ConceptList.svelte (hunk 1 above)
Comment (nsthorat) [reply web/blueprint/src/lib/components/commands/selectors/ConceptList.svelte]: Done

File: web/blueprint/src/lib/components/commands/CommandConcepts.svelte
Comment (nsthorat) [reply web/blueprint/src/lib/components/commands/CommandConcepts.svelte:1]: That random long number is because the video doesn't have auth. With auth you wont see those (because you cant compute over a dataset).

Regardless, I use a central util that will make those long numbers go away if we turn on computing concepts for auth.
